### DISCUSSION TITLE
Nomina dei primi ambassadors

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#38] [Annunci]
Come comunicato per vie traverse, abbiamo le prime 15 (QUINDICI) nomine degli ambassador del progetto.

In ordine di apparizione su Github nella lista dei membri del team, sono felicissimo di annunciare l'ingresso di @francescobianco, @eppak, @passasooz, @JeyDi, @guizzo, @davideimola, @serenasensini, @corradopetrelli, @nicolaerario, @elgorditosalsero, @imD3v che hanno già accettato l'invito e di Guido Penta, Simone Torrisi, Alessio D'Anna e Stefano Torresi che sono ancora pendenti!

Abbiamo già aperto una prima discussione per il primo incontro live: #37, pronti per partire?! Io non vedo l'ora!

**Comment by Cadienvan** [comment]
Aggiungo che @EMCestari , Emanuele Bartolesi e Claudio Bello sono entrati a far parte del team!

Let's rock and roll!

**Comment by guizzo** [comment]
@Cadienvan Direi che possiamo chiudere la discussione, non trovi?
